tests/Regression/GitHub/2725/BeforeAfterClassPidTest.php

```diff
@@ -32,6 +32,10 @@ public function testThatClassDidNotReload()
      */
     public static function showPidAfter()
     {
-        echo "\n@afterClass output - PID difference should be zero: " . ($GLOBALS['PID_BEFORE'] - getmypid());
```

Comment:
Please explain why an existing test case had to be modified.

Reply:
it has not,
logic here was kept the same,
only difference is that originally we saw only difference of PIDs, now we see their values, which helps debuggings

I agree that it's not nice to never display it, as you don't know if PIDs are same or they were not executed at all